src/ui/views/Player/RatingsOverview.basketball.tsx

```diff
@@ -1,67 +1,175 @@
 import PropTypes from "prop-types";
 import React from "react";
 import type { PlayerRatings } from "../../../common/types.basketball";
+import RatingWithChange from "../../components/RatingWithChange";
 
 const RatingsOverview = ({ ratings }: { ratings: PlayerRatings[] }) => {
 	const r = ratings.length - 1;
+	const lastSeason: PlayerRatings[] =
+		ratings.length > 1
+			? ratings.filter(s => s.season == ratings[r].season - 1)
+			: new Array(ratings[r]);
 
 	return (
 		<>
 			<div className="d-none d-lg-flex row">
 				<div className="col-lg-8">
-					<h2>Overall: {ratings[r].ovr}</h2>
+					<h2>
+						Overall:&nbsp;
```

Comment:
Unless there's a good reason for `&nbsp;` it shouldn't be used. You could use either `{" "}` or move the `<RatingWithChange` up to this line, like `Overall: <RatingWithChange`. Actually doesn't matter which you pick, I think Prettier will ultimately decide between them :)